lib/network/modules/components/nodes/util/CircleImageBase.js

```diff
@@ -24,17 +58,21 @@ class CircleImageBase extends NodeBase {
   }
 
   /**
-   * Switch between the base and the selected image.
+   * Set selection and switch between the base and the selected image.
+   *
+   * Do the switch only if imageObjAlt exists.
+   *
+   * @param {true|false} selected value of new selected state for current node
    */
   switchImages(selected) {
```

Comment:
For better naming: what is actually happening here? If the node is selected, the selected-image is set to be displayed instead of the regular image. So perhaps `setSelectedImage()`.

But I think the naming is good enough as it is. Leaving it like this.